.github/workflows/deploy-staging.yaml

```diff
@@ -0,0 +1,58 @@
+name: Deploy to staging
+
+on:
+  push:
+    pull_request:
+      types: [ ready_for_review, opened, reopened, synchronize ]
+      branches:
+        - main
+  # Allows you to run the workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [ zookeeper-1, zookeeper-2, kafka-1, kafka-2 ]
+
+    name: Deploy ${{ matrix.app }}
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup gcloud CLI for deploy
+        uses: google-github-actions/auth@v0
+        with:
+          credentials_json: ${{ secrets.DIGDIR_FDK_DEV_AUTODEPLOY }}
+
+      - name: Apply Helm template
+        run: |
+          helm template \
+          ./helm-sources/${{ matrix.app }}/ > ./kubectlapply-${{ matrix.app }}.yaml \
+          -f ./helm-sources/${{ matrix.app }}/values/staging.yaml 
+          chmod o+w ./kubectlapply-${{ matrix.app }}.yaml
+          cat kubectlapply-${{ matrix.app }}.yaml
+
+      - name: Deploy to GCP
+        run: |
+          gcloud container clusters get-credentials digdir-fdk-dev --region europe-north1-a --project digdir-fdk-dev
+          kubectl apply -f ./kubectlapply-${{ matrix.app }}.yaml --force
+          kubectl --namespace=staging rollout restart statefulset ${{ matrix.app }}
+
+
+
+
+
+#      - name: Notify slack
```

Comment:
Huske å kommentere inn igjen. For å få med evt fails, så burde det kanskje gå en slack-melding pr matrix app.